WebRoot/js/ajax/config/msgs/AjxMsg_vi.properties

```diff
@@ -0,0 +1,157 @@
+# 
```

Comment:
It was, sorry. 
I removed it and rebased. I only see the 3 correct files now. Hope you do too.
https://github.com/Zimbra/zm-ajax/pull/34/files